requirements.txt

```diff
@@ -1,4 +1,4 @@
 click
 requests
 terminaltables
-PyGithub==1.43.7
+PyGithub
```

Comment:
I needed to rm the pinned version number here to avoid the following error:
```
error in PyGithub setup command: use_2to3 is invalid
```

Reply:
is it possible something is not working with your local python setup?